### PR TITLE
Feature/social network v2

### LINKED ISSUE #5: Include sharing links
When clicking on share, a grid with 4 items is displayed.

The proposal is to add the respective links and images (preferably in `.svg`) for sharing on social networks.

Grid Share

---

I usually find the `.svg` I need here: https://www.svgrepo.com/

### PR DESCRIPTION
**Pull request by hermannleboss**
I delet the old branch and rebase develop on the new one.

Close #5 

I try it here.

### CODE REVIEW

**Comment by arnonrdp** [comment]
@hermannleboss, I didn't understand why you close the #12. You could have just done a `git merge staging` from your feature branch. It would update your branch. But ok.

Can you enhance the function so that it is also possible to share over the web as well.

File: src/pages/PromptPage.vue [hunk 1]
```diff
@@ -73,6 +73,24 @@ onMounted(async () => {
...
+async function onShare() {
+  const title = document.title
+  const url = document.querySelector('link[rel=canonical]') ? document.querySelector('link[rel=canonical]').href : document.location.href
+  const text = 'I find this usefully'
+  try {
+    await navigator.share({
+      title,
+      url,
+      text
+    })
```
Comment (hermannleboss) [inline src/pages/PromptPage.vue:85]: Hello @arnonrdp 
I'm using here native navigation function to be able to share. 
This function is only available in secure contexts

[the doc](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share)

image

**Comment by hermannleboss** [comment]
@arnonrdp and for enchancement, 
two possible way,

Firt : Use native share on mobile and social share for computer.

File: src/pages/PromptPage.vue (hunk 1 above)
Comment (arnonrdp) [reply src/pages/PromptPage.vue:85]: Noted the need of https, it is alright.

**Comment by arnonrdp** [comment]
> @arnonrdp and for enchancement, two possible way,
> 
> Firt : Use native share on mobile and social share for computer.

Are you developing this?

**Comment by hermannleboss** [comment]
> > @arnonrdp and for enchancement, two possible way,
> > Firt : Use native share on mobile and social share for computer.
> 
> Are you developing this?

No i think, we can resolve this first and then 
Create a new PR / Issuel for computer version.

**Pull request closed without merging**